### ISSUE TITLE
Doesn't show detailed error message from golangci-lint

### ISSUE THREAD

**Issue by BorzdeG** [xxpxxxxp/intellij-plugin-golangci-lint#119]
<img width="406" alt="image" src="https://user-images.githubusercontent.com/2332528/185050856-7b662b02-4db9-4e17-8aa0-ee3bca578de5.png">

in GoLand log:
```
2022-08-17 04:46:01,155 [33937015]   WARN - go-linter - Run error: level=warning msg="[linters context] contextcheck is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649."
level=warning msg="[runner] Can't run linter goanalysis_metalinter: goimports: can't extract issues from gofmt diff output \"--- $PROJECT_DIR$/parser/firebase-parser.go.orig\\t2022-08-17 04:46:01\\n+++ $PROJECT_DIR$/parser/firebase-parser.go\\t2022-08-17 04:46:01\\n@@ -3,9 +3,10 @@\\n import (\\n \\t\\\"context\\\"\\n \\n+\\t\\\"PKG_HIDDEN/jwt/auth/model\\\"\\n+\\n \\t\\\"firebase.google.com/go/auth\\\"\\n \\t\\\"github.com/pereslava/grpc_zerolog/ctxzerolog\\\"\\n-\\t\\\"hyperflex/go/shared/jwt/auth/model\\\"\\n )\\n \\n type FirebaseParser interface {\\n\": can't parse patch: parsing time \"2022-08-17 04:46:01\" as \"2006-01-02 15:04:05 -0700\": cannot parse \"\" as \"-0700\""
```

`$PROJECT_DIR$` and `PKG_HIDDEN` - private data cut by me

**Comment by xxpxxxxp** [comment]
Hi @BorzdeG , you mean you'd like to see detailed error info in such panel?

**Comment by BorzdeG** [comment]
Yes. So that you do not have to go to the IDE log and see why the error occurred.

With your permission, I'll try to add this feature myself.

**Comment by mikeschinkel** [comment]
@BorzdeG — Any status on this?  

I just asked about how to find the error longs in the [#Gland channel](https://gophers.slack.com/archives/C3FJ8M2PN/p1685394373346739) of Gopher's Slack and was pointed to this issue.

I would love to help if you haven't gotten to it, but thus far I have been completely thwarted to write plugins due to my lack of of Java/Kotlin experience.

**Comment by BorzdeG** [comment]
@mikeschinkel No, I haven't gotten to it yet :(
